### PR TITLE
Add Circle Linux 9 configs

### PR DESCRIPTION
**Pull request by bella485**
Circle Linux 9.3 released, now support x86_64 aarch64 ppc64le s390x.

### CODE REVIEW

**Comment by bella485** [comment]
Hi maintainers,   Happy new year !

Circle Linux 9 had published  the fourth release Circle Linux 9.3 https://mirror.cclinux.org/pub/circle/  ,  we now building 8.9 because we carry the host to another place, we sorry to late.

Please let me know if there is any problem.

File: mock-core-configs/etc/mock/templates/circlelinux-9.tpl
Comment (praiskup) [inline mock-core-configs/etc/mock/templates/circlelinux-9.tpl:6]: $ podman run --rm -ti quay.io/cclinux/circlelinux:9 bash
Trying to pull quay.io/cclinux/circlelinux:9...
Error: initializing source docker://quay.io/cclinux/circlelinux:9: reading manifest 9 in quay.io/cclinux/circlelinux: manifest unknown

**Comment by praiskup** [comment]
Thank you for working on this.

**Comment by praiskup** [comment]
@bella485 note we plan the release #1307 - so gently pinging; it would be nice to update the PR ASAP so we can ship it.

**Comment by praiskup** [comment]
Last gently ping, but I think we can wait for the next mock-core-configs release.

**Review by praiskup — CHANGES_REQUESTED** [review]
Some changes are needed, either we should disable bootstrap image, or fix the image location.

File: mock-core-configs/etc/mock/templates/circlelinux-9.tpl
Comment (bella485) [reply mock-core-configs/etc/mock/templates/circlelinux-9.tpl:6]: Hey , sorry too late , I just had a long holiday for the Chinese New Year.  now we had pushed podman images to https://quay.io/repository/cclinux/circlelinux ;

**Comment by bella485** [comment]
> Some changes are needed, either we should disable bootstrap image, or fix the image location.

we had pushed podman images to https://quay.io/repository/cclinux/circlelinux；

 podman run --rm -ti quay.io/cclinux/circlelinux:9 bash

work now ， thanks a lot；

**Comment by bella485** [comment]
> Some changes are needed, either we should disable bootstrap image, or fix the image location.

Hi brother,  thanks a lot your work review for circlelinux mock config, please help me merge it , thanks again .
good luck ~

**Comment by praiskup** [comment]
@bella485 have you removed the branch/repo intentionally?

**Comment by bella485** [comment]
yes,I have sync fork the upstream project,add pull request again.Sorry to bother you.

**Pull request closed without merging**
